client/types/cbac.go

```diff
@@ -101,7 +101,7 @@ const (
 	UsersAndGroupsCat  = `Users and Groups`
 	SystemAndStatsCat  = `System and Stats`
 	SecretsCat         = `Secrets`
-	RemoteAIServiceCat = `Remote AI Service APIs`
+	LogbotAIServiceCat = `Logbot AI Service APIs`
```

Comment:
Can we just say "Logbot AI" ? I don't think it adds value to mention APIs here.

Reply:
agreed, fixed